### PR TITLE
Expand codeowner teams

### PR DESCRIPTION
**Pull request by raman325**
As discussed, this updates the codeowners checks to expand teams into user lists

### CODE REVIEW

File: services/bots/src/github-webhook/utils/teams.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { Organization } from '../github-webhook.const';
+import { GithubClient } from '../github-webhook.model';
+
+const ORG_TEAM_SEP = '/';
+const TEAM_PREFIX = `${Organization.HOME_ASSISTANT}${ORG_TEAM_SEP}`;
+
+export const expandTeams = async (
```
Comment (ludeeus) [inline services/bots/src/github-webhook/utils/teams.ts:7]: I think we should extend [`WebhookContext`](https://github.com/home-assistant/service-hub/blob/main/services/bots/src/github-webhook/github-webhook.model.ts#L36) with this function and change the name to `expandOrganizationTeams`

File: services/bots/src/github-webhook/utils/teams.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  // Initialize list with users from usersAndTeams
+  const users = usersAndTeams.filter((name) => !name.startsWith(TEAM_PREFIX));
+  // For each team in usersAndTeams, add the members of the team to the list
+  for (const team in usersAndTeams
+    .filter((name) => !users.includes(name))
+    .map((name) => name.split(ORG_TEAM_SEP)[1])) {
+    users.push(
+      ...(
+        await github.teams.listMembersInOrg({
+          org: Organization.HOME_ASSISTANT,
```
Comment (ludeeus) [inline services/bots/src/github-webhook/utils/teams.ts:24]: Once moved, use `org: this.organization,` here

File: services/bots/src/github-webhook/utils/teams.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+export const expandTeams = async (
+  usersAndTeams: string[],
+  github: GithubClient,
+): Promise<string[]> => {
+  // Remove the `@` and lowercase
+  usersAndTeams = usersAndTeams.map((name) =>
+    name.startsWith('@') ? name.substring(1).toLowerCase() : name.toLowerCase(),
+  );
+  // Initialize list with users from usersAndTeams
+  const users = usersAndTeams.filter((name) => !name.startsWith(TEAM_PREFIX));
```
Comment (ludeeus) [inline services/bots/src/github-webhook/utils/teams.ts:16]: This would need an adjustment to use ``${this.organization}${ORG_TEAM_SEP}``

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: services/bots/src/github-webhook/utils/teams.ts (hunk 1 above)
Comment (ludeeus) [reply services/bots/src/github-webhook/utils/teams.ts]: Extend was my initial thought, but if we for the most part pass it, we can continue with that here.

**Comment by raman325** [comment]
OK done with tweaks, realized I could simplify things

**Comment by raman325** [comment]
OK **NOW** I'm done 😅

**Review by ludeeus — APPROVED** [review]
Thanks!

**Pull request merged**